webauthn/webauthn.py

```diff
@@ -85,9 +85,11 @@ class WebAuthnUserDataMissing(Exception):
 class WebAuthnMakeCredentialOptions(object):
 
     _attestation_forms = {'none', 'indirect', 'direct'}
+    _userVerification = {'required','preferred','discouraged'}
```

Comment:
Minor nit: can you put spaces after the commas?

`_userVerification = {'required', 'preferred', 'discouraged'}`